internal/api/list_images.go

```diff
@@ -0,0 +1,72 @@
+package api
```

Comment:
Just to stick with the current naming convention for files in this package, the filename should probably be `list_images_v1alpha1`.

Reply:
This is my only feedback. Pre-emptively LGTM'ing.

Reply:
🤦🏻 my intention was to name it that way... didn't have enough coffee yesterday 😆